elastic/eui
# feat: add Loki

## Summary

This PR resolves #7616 by setting up Loki to test visual regressions based on storybook stories.

I recommend reviewing commit by commit and skipping through `chore: add reference snapshots.`

## QA

1. Checkout this branch locally (`gh pr checkout 7679`)
2. Install new dependencies (`yarn`) and run storybook locally (`yarn storybook --no-open`)
3. [ ] Run `yarn test-visual-regression`, wait till it ends testing, and confirm there are no failed tests reported
